v2/speechtotextv1/speech_to_text_v1_integration_test.go

```diff
@@ -404,11 +405,17 @@ func (cb myCallBack) OnClose() {
 func (cb myCallBack) OnData(resp *core.DetailedResponse) {
 	var speechResults speechtotextv1.SpeechRecognitionResults
 	result := resp.GetResult().([]byte)
+
+	fmt.Println(resp)
```

Comment:
Do we want these print statements here? As well as the fmt import?

Reply:
Nice catch I will remove

Reply:
removed